requirements.txt

```diff
@@ -58,3 +58,4 @@ tqdm==4.53.0              # via mesa
 uncertainties==3.1.5      # via time-series-buffer
 urllib3==1.26.2           # via requests
 werkzeug==1.0.1           # via flask
+visdcc==0.0.40
```

Comment:
We need to add this dependency to `setup.py` as well.

Reply:
yes, the visdcc is quite a temporary fix to launch some javascript code callbacks (which was also done with the mpld3 without visdcc, but with a bit more implementation). ideally we wont need the visdcc later on, and just use the built-in dash methods to do the same (like what we did with mpld3).

Reply:
Ah, I see, so anyway we will not release anything with this in the requirements, right? Then we do not need to add it to `setup.py`.